src/main/java/org/openjdk/jextract/impl/FunctionalInterfaceBuilder.java

```diff
@@ -38,28 +37,29 @@
 
 final class FunctionalInterfaceBuilder extends ClassSourceBuilder {
 
+    private static final String MEMBER_MODS = "static";
```

Comment:
Seems unused? (I think I prefer just 'inlining' the mods in the string template if they don't change, as you've done)